spec/GraphQLOverHTTP.md

```diff
@@ -178,49 +227,74 @@ A request for execution should contain the following request parameters:
 - {extensions} - (_Optional_): This entry is reserved for implementors to extend
   the protocol however they see fit.
 
-Note: Be aware that `query` is a misleading name as it can contain a string
-describing multiple operations, each of which may be a query, mutation or
-subscription. A better name would have been `document`, but the term `query` is
-well established.
+Note: When comparing {GraphQL-over-HTTP request} against the term
+["request"](https://spec.graphql.org/draft/#request) in the GraphQL
+specification you should note the {GraphQL schema} and "initial value" are not
+included in the GraphQL-over-HTTP {request}, they are handled by the {server}
+based on the URL used.
+
+Note: Be aware that `query` is a misleading parameter name as its value is a
+string describing one or more operations, each of which may be a query, mutation
+or subscription. A better name would have been `document`, but the term `query`
```

Comment:
This is the only reference to subscriptions in the spec. Perhaps it should be dropped, and/or there should be an explicit note that this spec doesn't provide a transport for subscriptions at this time.